python/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.10.10-slim
```

Comment:
can you use [3.10.10-alpine3.16](https://hub.docker.com/layers/library/python/3.10.10-alpine3.16/images/sha256-98b88b3b8e2c3d889b28d7af828635bcbb6c0242960930c5e399e2b7bd47ff96?context=explore) since in all other Dockerfiles we use alpine?